lua/includes/modules/minigames.lua

```diff
@@ -89,11 +89,15 @@ local function SetupData(minigame)
 
 	---
 	-- @name ttt2_minigames_[MINIGAME]_enabled
-	CreateConVar("ttt2_minigames_" .. minigame.name .. "_enabled", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED})
+	if not ConVarExists("ttt2_minigames_" .. minigame.name .. "_enabled") then
```

Comment:
This check isn't necessary, if it already exists, the existing cvar will be returned (no new cvar is created).